lib/spack/docs/known_issues.rst

```diff
@@ -0,0 +1,16 @@
+============
+Known issues
+============
+
+Below is the list of known issues:
+
+#. Current conretization algorithm does not take into account default values
```

Comment:
Oh, and add a link to the Issue that first reported the problem so that people can look there for more information if they are curious. For this issue, it is #397.